lighthouse-core/lib/dependency-graph/simulator/simulator.js

```diff
@@ -405,9 +431,20 @@ class Simulator {
 
     return {
       timeInMs: totalElapsedTime,
-      nodeTimings: this._nodeTimings,
+      nodeTimings: this._computeFinalNodeTimings(),
```

Comment:
since we're now emitting a diff shape at the end should we use diff property names to keep totally clear?   nodeTimeSpans  vs nodeTimings? eh

Reply:
I would prefer to rename the internal since that's the one that carries more than just timings data

Reply:
Which name would you prefer since you were the original ✍️ of `nodeTiming` ;) https://github.com/GoogleChrome/lighthouse/pull/3187#discussion_r136454948